### PR TITLE
Update the copy on email templates

### PR DESCRIPTION
**Pull request by adam-thomas**
* Don't pretend the links are clickable when they aren't.
* Let the user know they can reply to the email itself.
* Add some quotes to aid readability.

### CODE REVIEW

**Comment by adam-thomas** [comment]
@incuna/backend review please? :)

File: groups/templates/groups/emails/new_discussion.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {% load i18n %}{% blocktrans with username=user.get_full_name name=discussion.name group=discussion.group.name creator=discussion.creator.name %}Dear {{ username }},
 
-A new discussion "{{ name }}" has been posted to the group {{ group }} by {{ creator }}{% endblocktrans %}{% if discussion.comments.first.body %}:
+A new discussion on "{{ name }}" has been posted to the {{ group }} group by {{ creator }}{% endblocktrans %}{% if discussion.comments.first.body %}:
```
Comment (meshy) [inline groups/templates/groups/emails/new_discussion.txt]: Should this be `the "{{ group }}" group`?

**Comment by meshy** [comment]
Other than my question, LGTM

File: groups/templates/groups/emails/new_discussion.txt (hunk 1 above)
Comment (adam-thomas) [inline groups/templates/groups/emails/new_discussion.txt]: I thought about that, but I think groups will have names that fit this pattern well enough that it should work out.  `...been posted to the Haematology group`, that sort of thing.  I'm happy to add the quotes if you reckon they'll be needed though :)

Comment (meshy) [inline groups/templates/groups/emails/new_discussion.txt]: I'm happy to trust your judgement on this.

**Comment by meshy** [comment]
:cake:

**Comment by adam-thomas** [comment]
Thanks :)

**Pull request merged**